node/src/node_configurator/mod.rs

```diff
@@ -382,14 +203,6 @@ pub fn request_existing_db_password(
     Ok(result)
 }
 
-pub fn cannot_be_blank(password: &str) -> Result<(), String> {
-    if password.is_empty() {
-        Err("Password cannot be blank.".to_string())
-    } else {
-        Ok(())
-    }
-}
-
 #[derive(Debug, PartialEq, Clone)]
 pub enum PasswordError {
     Mismatch,
```

Comment:
The life of these all (probably) also depends on your decision about keeping or deleting request_existing_db_password